.github/actions/send-telegram-notify/action.yml

```diff
@@ -28,14 +28,59 @@ runs:
         uname -s | tee -a $GITHUB_ENV
         echo 'EOF' >> $GITHUB_ENV
       shell: bash
+    # check if host OS is rhel or fedora
+    - run: |
+        echo 'OS_IS_FEDORA_OR_RHEL<<EOF' >> $GITHUB_ENV
+        grep "^ID=" /etc/os-release | grep -e 'centos' -e 'fedora' && echo true | tee -a $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+      shell: bash
     # get job id number
     - run: |
-        apt update ||:
-        apt install -y jq ||:
+        sudo -n apt update -y ||:
+        sudo -n apt install -y jq ||:
         echo 'JOB_ID<<EOF' >> $GITHUB_ENV
         curl -s https://api.github.com/repos/tarantool/tarantool/actions/runs/${{ github.run_id }}/jobs | jq -r '.jobs[0].id' | tee -a $GITHUB_ENV
         echo 'EOF' >> $GITHUB_ENV
       shell: bash
+    # convert commit message to single line with marked new lines as 'TELEGRAMNEWLINE'
+    - env:
+        COMMIT_MSG: |
+          ${{ github.event.head_commit.message }}
+      run: |
+        # Make changes in the commit message to avoid of its fail on Markdown:
+        # 1. Convert message from multi lines to single line.
+        # 2. Add backslashes to single quote marks in message.
+        # 3. Add backslashes to '`' in message.
+        if ${{ env.OS_IS_FEDORA_OR_RHEL == 'true' }} || ${{ env.KERNEL_NAME == 'Darwin' }} ; then
+          msg="${COMMIT_MSG//$'\n'/TELEGRAMNEWLINE}"
+          msg="${msg//$'\\\\'/BACKSLASH}"
+          msg="${msg//$'\\\''/\'}"
+          msg="${msg//$'`'/BACKTICK}"
+        else
+          msg="${COMMIT_MSG//$'\n'/TELEGRAMNEWLINE}"
+          msg="${msg//$'\\'/BACKSLASH}"
+          msg="${msg//$'\''/\\\'}"
+          msg="${msg//$'`'/BACKTICK}"
+        fi
+        echo 'COMMIT_MSG_LINE<<EOF' >> $GITHUB_ENV
+        echo "$msg" | tee -a $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+      shell: bash
+    # strip Markdown from commit message
+    - run: |
+        pip install setuptools
+        pip install markdown bs4
```

Comment:
Should we use here pip3?

Reply:
If a box couldn't contain a python2 and the link couldn't be a link to pip2 it is ok. Otherwise, please rename.